src/actionCreators/index.js

```diff
@@ -21,16 +21,25 @@
 import axios from 'axios';
 import {
   authorizationStart,
+  authorizationPending,
   authorizationSuccess,
   authorizationFailed,
   authorizationError,
 } from '../actions';
 import { API } from '../constants';
+
 const axi = axios.create({
   baseURL: API.BASE_URL(),
   timeout: API.TIME_OUT,
 });
 
+const checkStatus = (isPending = false, isAuthorized = false, isRejected = false) => {
+  if (isRejected) return 3;
```

Comment:
is this meant to have a priority? In the fact that if all are true, isRejected returns its value?

Reply:
will be refactoring soon!